### DISCUSSION TITLE
Issue with Adjusting Width of Publication Elements

### DISCUSSION THREAD

**Discussion by ZJ-Gao** [alshedivat/al-folio#2619] [Q&A] [answered]
Hi everyone,

I'm currently working on modifying the layout of the publication elements on [my site.](https://zj-gao.github.io/zj_web/) I've been able to successfully adjust the width of the publication title, ensuring it doesn't break into a new line after just a few words. This adjustment has made the title display much more cleanly and consistently.

However, I've encountered issues when trying to apply the same method to the author line and the abstract line. Despite my efforts, these elements still default to a width of 570px, while I intend for them to be 700px, similar to the adjusted title.

Here's a summary of what I've tried so far:

    Changes in bib.liquid:
        I applied CSS classes to set the width of the author and abstract lines to 80% of the container, with a maximum width of 720px, and used margin-left: auto; and margin-right: auto; to center them.
        I also ensured to add whitespace control in Liquid template for the author section to keep the HTML clean.

    Adjustments in _base.scss and layout.scss:
        I attempted to set the desired widths directly in these SCSS files but found no change in the rendered output.

Despite these efforts, the width of the author and abstract lines remains unchanged. The attached images illustrate the current state, where the width is still 570px instead of the intended 700px.

I would appreciate any guidance or suggestions on how to resolve this issue. Specifically, I'm looking for advice on:

    Correctly targeting and adjusting the width of the author and abstract lines in the appropriate files.
    Ensuring that the custom styles take precedence over default or inherited styles.

Thank you for your help!

Attachments:

    Current state showing the author and abstract lines at 570px width.
<img width="538" alt="image" src="https://github.com/user-attachments/assets/f253305a-9231-49d6-82a4-e513071c4085">
<img width="559" alt="image" src="https://github.com/user-attachments/assets/131da5d9-981b-45b4-bc00-093f5d0360ec">

**Comment by george-gca** [comment, marked as answer]
I made a small test in our [template site](https://alshedivat.github.io/al-folio/publications/) and it seems to work as expected. Change `col-sm-8` to `col-sm-9` here:

https://github.com/alshedivat/al-folio/blob/1d4ce5a313d1c41e73c843587692eabebad96e00/_layouts/bib.liquid#L48

**Reply by ZJ-Gao** [reply]
That worked and thanks! I had another issue that my author line exceeded the container's length (too many authors), which might be because I changed some setup somewhere else. I added the following CSS on top of the `bib.liquid` file to fix it: 
`<style>
  .publication-item {
    word-wrap: break-word;
  }
</style>` 
on top of the bib.liquid and it works.
